sparse/usr/bin/droid/android-links.sh

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env bash
 
-MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)
+MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)/media/0
+
+#fix a stupid mistake in previous versions
+[[ "$(readlink -- /home/nemo/Pictures/Android)" == "/android/Pictures" ]] && unlink /home/nemo/Pictures/Android || echo "clean"
+unlink /home/nemo/Music/Android
+unlink /home/nemo/Playlists/Android
```

Comment:
this is not an if-block, and now these lines will get unliked every time (means on 2nd run script will break)